### ISSUE TITLE
Access Violation Exception Occurs While Opening and Scrolling Through a ComboBox

### ISSUE THREAD

**Issue by jrovny** [microsoft/microsoft-ui-xaml#7556]
### Describe the bug

We encountered an exception when attempting to open and scroll through (using the scrollbar) a combo box.

Interestingly, when we switch the Microsoft.WindowsAppSDK version to 1.0.4 the issue doesn't occur. The issue only occurs starting with Microsoft.WindowsAppSDK version 1.1.0 and up to 1.1.3.

Possibly related to issue https://github.com/microsoft/WindowsAppSDK/issues/2712?

The issue doesn't seem to occur with list boxes and it doesn't occur when scrolling using a mouse scroll wheel. You have to actually select the vertical scrollbar and attempt to scroll using the scrollbar.

Here's the raw exception message:
Exception thrown at 0x00007FFEA949C62E (ntdll.dll) in SampleAccessViolationExceptionApp.exe: 0xC0000005: Access violation reading location 0xFFFFFFFFFFFFFFFF.

### Steps to reproduce the bug

1. Create a new "Blank App, Packaged (WinUI 3 in Desktop)" project 
3. Populate the combo box with a large number of items (at least 5,000) in a simple loop (it doesn't take nearly as many items if loaded from a datastore asynchronously)
4. Open the combo box, select the vertical scrollbar in the combo box and attempt to scroll in either direction
5. Exception should occur

```
public MainWindow()
{
    this.InitializeComponent();

    // Example code to populate combo box
    for (int i = 0; i < 5000; i++)
    {
        this.valuesComboBox.Items.Add(new Value { Title = $"This is test {i}" });
    }

    this.valuesComboBox.SelectedIndex = 500;
}
```

### Screenshots

![image](https://user-images.githubusercontent.com/13325610/182208062-74678f76-0cfa-43d3-bfb8-21e7c056d9d4.png)

### NuGet package version

1.1.3

### Packaging type

Packaged (MSIX)

### Windows version

Windows 10 version 1909 (18363, November 2019 Update)

### IDE

Visual Studio 2022

**Comment by castorix** [comment]
I had posted a similar test in this issue [#7101](https://github.com/microsoft/microsoft-ui-xaml/issues/7101)
by simply filling the ComboBox with a loop

**Comment by jrovny** [comment]
@castorix yep, looks like it's the same issue. This can probably be listed as a duplicate of #7101 .

**Comment by jrovny** [comment]
This issue was resolved as part of release 1.1.4. Thanks for the help!
